mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go

```diff
@@ -0,0 +1,73 @@
+package securitycenterv2_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccSecurityCenterV2OrganizationMuteConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterV2OrganizationMuteConfig_basic(context),
+			},
+			{
+				ResourceName:      "google_scc_v2_organization_mute_config.default",
```

Comment:
Discussed offline, import is failing because we made bad assumptions about `self_link` (we append `name`) and an explicit `import_format` is required.

Reply:
Fixed.